sentry-ruby/lib/sentry/hub.rb

```diff
@@ -229,6 +229,52 @@ def with_session_tracking(&block)
       end_session
     end
 
+    def get_traceparent
+      return nil unless current_scope
+
+      current_scope.get_span&.to_sentry_trace ||
+        current_scope.propagation_context&.get_traceparent
```

Comment:
```suggestion
      current_scope&.get_span&.to_sentry_trace ||
        current_scope&.propagation_context&.get_traceparent
```
I believe this suggestion produces the same result more compactly